<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>farmaceutico_modificar_medicamentos</class>
 <widget class="QDialog" name="farmaceutico_modificar_medicamentos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: white;</string>
  </property>
  <widget class="QLabel" name="nombre_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Medicamento</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Tipo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="text">
    <string>Descripcion:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="text">
    <string>Costo: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="text">
    <string>Premedicado:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="descripcion_textEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 5px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tipo_lineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border: none;
border-bottom: 1px solid gray;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="costo_lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border: none;
border-bottom: 1px solid gray;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="premedicado_lineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border: none;
border-bottom: 1px solid gray;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_modificar_medicamento">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color: white;
	color: #3E64FF;
	border: 1px solid #3E64FF;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: #3E64FF;
}</string>
   </property>
   <property name="text">
    <string>Modificar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancelar_modificacion">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>370</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color: #EF1F4B;
	color: white;
	border: 1px solid #EF1F4B;
	border-radius: 5px;
	padding: 5px;
}
</string>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
